packages/__examples__/.storybook/stories/stories.ts

```diff
@@ -22,18 +22,61 @@
  * SOFTWARE.
  */
 
+import { ComponentType } from 'react'
+
 import { storiesOf } from '@storybook/react'
-import { renderExample } from './renderExample'
-import { renderPage } from './renderPage'
+import type { LegacyStoryFn, StoryKind, StoryName } from '@storybook/addons'
+import type { StoryFnReactReturnType } from '@storybook/react/dist/ts3.9/client/preview/types'
 import {
   generateComponentExamples,
   StoryConfig
 } from '@instructure/ui-test-utils'
-import { ComponentType } from 'react'
+
+import { renderExample } from './renderExample'
+import { renderPage } from './renderPage'
+
 // must be imported with Webpack because this file cannot contain async calls
 // @ts-ignore TODO figure out why this is an error
 import propJSONData from '../../prop-data.json'
+
 import TooltipPositioning from './TooltipPositioning'
+// import SourceCodeEditorExamples from './SourceCodeEditorExamples'
+
+type AdditionalExample = {
+  title: StoryKind
+  stories: {
+    storyName: StoryName
+    storyFn: LegacyStoryFn<StoryFnReactReturnType>
+    chromaticSettings?: Record<string, any>
+  }[]
+}
+
+// Stories not tied to one component and custom component examples
+const additionalExamples: AdditionalExample[] = [
+  {
+    title: 'Tooltip positioning',
+    stories: [
+      {
+        storyName: 'Tooltip positions',
+        storyFn: () => TooltipPositioning()
+      }
+    ]
+  }
+  // TODO: try to fix the editor not rendering fully on chromatic screenshot,
+  //  even with delay
+  // {
+  //   title: 'SourceCodeEditor',
+  //   stories: [
+  //     {
+  //       storyName: 'Examples',
+  //       storyFn: () => SourceCodeEditorExamples(),
+  //       chromaticSettings: {
+  //         delay: 3000
+  //       }
+  //     }
+  //   ]
+  // }
```

Comment:
While these do not work perfect I'd still add them

Reply:
Ok, I'll add them back, but left the TODO there, maybe it can be fixed later.